### DISCUSSION TITLE
Confirmation prompt for tar_destroy()

### DISCUSSION THREAD

**Discussion by gofford** [ropensci/targets#542] [Ideas]
`targets` is an excellent framework, and has reduced code-complexity in my projects immeasurably.

However, the `tar_destroy()` functions feels very heavy handed at the moment. It is single-use, and non-recoverable. If I (or a collaborator) accidentally types it our entire artefact history is gone.

Have you considered adding a confirmation prompt to this function? I.e., `tar_destroy()` -> "Are you sure?" -> Yes.

**Comment by wlandau** [comment]
A prompt is a good idea, added in e46f178d6b71c90a99dc5c89d89512fc3012fd20. And there are several cleaning functions that are less extreme than `tar_destroy()`: https://docs.ropensci.org/targets/reference/index.html#section-clean
